src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
```

Comment:
Please make code more readable
```suggestion
  const women = withChildren
    ? people.filter((p) => {
       return (p.sex === 'f'
        && people.find((p1) => {
            return p1.mother === p.name;
           }))
      })
    : people.filter((p) => {
        return p.sex === 'f';
      });
```